gpu/test_gpu.py

```diff
@@ -120,11 +120,14 @@ def test_install_gpu_cuda_nvidia(self, configuration, machine_suffixes,
                                    master_accelerator, worker_accelerator,
                                    cuda_version):
 
+    if self.getImageOs() == "debian" and self.getImageVersion() >= pkg_resources.parse_version("2.2"):
```

Comment:
Can we ensure this test to be skipped based on cuda version rather than dataproc version.

Reply:
Thank you!  Testing fix now.

Reply:
Anything further?  I think that with this change, I can release the lock for a bit so that you can get the Kerberos changes in.

Reply:
Looks good.